docs/advanced_usage/9_parallelism.md

```diff
@@ -1,41 +1,76 @@
-    overhead of communcation. Consider using only one worker in this case.
-
+    When using multiple workers, SMAC is not reproducible anymore.
 
-!!! warning
 
-    When using multiple workers, SMAC is not reproducible anymore.
+## Parallelizing Locally
 
+To utilize parallelism locally, that means running workers on the same machine as the main jobs, specify the ``n_workers`` keyword when creating the scenario.
+```python
+Scenario(model.configspace, n_workers=5)
+``` 
 
-## Running on a Cluster
 
-You can also pass a custom dask client, e.g. to run on a slurm cluster.
-See our [parallelism example](../examples/1%20Basics/7_parallelization_cluster.md).
+## Parallelizing on SLURM
 
-!!! warning
+To utilize this split of main and execution jobs on a [SLURM cluster](https://slurm.schedmd.com/), SMAC supports manually specifying a [Dask](https://www.dask.org/) client.
+This allows executing the target function on dedicated SLURM jobs that are necessarily configured with the same hardware requirements,.
 
-    On some clusters you cannot spawn new jobs when running a SLURMCluster inside a
-    job instead of on the login node. No obvious errors might be raised but it can hang silently.
+!!! note
 
-!!! warning
+    While most SLURM clusters behave similarly, the example DASK client, might not work for every cluster. For example, some clusters only allow spawning new jobs 
+    from the login node.
 
-    Sometimes you need to modify your launch command which can be done with
-    `SLURMCluster.job_class.submit_command`.    
+To configure SMAC properly for each cluster, you need to know the ports which allow communication between main and worker jobs. The dask client is then created as follows:
 
 ```python
-cluster.job_cls.submit_command = submit_command
-cluster.job_cls.cancel_command = cancel_command
-```
\ No newline at end of file
+...
+from smac import BlackBoxFacade, Scenario
+from dask_jobqueue import SLURMCluster
+
+cluster = SLURMCluster(
+    queue="partition_name",                 # Name of the partition
+    cores=4,                                # CPU cores requested
+    memory="4 GB",                          # RAM requested
+    walltime="00:10:00",                    # Walltime limit for a runner job. 
+    processes=1,                            # Number of processes per worker
+    log_directory="tmp/smac_dask_slurm",    # Logging directory
+    nanny=False,                            # False unless you want to use pynisher
+    worker_extra_args=[
+        "--worker-port",                    # Worker port range 
+        "60010:60100"],                     # Worker port range 
+    scheduler_options={
+        "port": 60001,                      # Main Job Port
+    },
+)
+cluster.scale(jobs=n_workers)
+
+# Dask creates n_workers jobs on the cluster which stay open.
+client = Client(
+    address=cluster,
+)
+
+#Dask waits for n_workers workers to be created
+client.wait_for_workers(n_workers)
```

Comment:
ahh okay so you need to start the client before initializing smac 😵

Reply:
:D
It needs to be done, before smac.optimize is called. But this looked more easily understood to me